lib/cli-engine/formatters/formatters-meta.json

```diff
@@ -1,46 +1,18 @@
 [
-    {
-        "name": "checkstyle",
-        "description": "Outputs results to the [Checkstyle](https://checkstyle.sourceforge.io/) format."
-    },
-    {
-        "name": "compact",
-        "description": "Human-readable output format. Mimics the default output of JSHint."
-    },
     {
         "name": "html",
         "description": "Outputs results to HTML. The `html` formatter is useful for visual presentation in the browser."
     },
-    {
-        "name": "jslint-xml",
-        "description": "Outputs results to format compatible with the [JSLint Jenkins plugin](https://plugins.jenkins.io/jslint/)."
-    },
-    {
-        "name": "json-with-metadata",
-        "description": "Outputs JSON-serialized results. The `json-with-metadata` provides the same linting results as the [`json`](#json) formatter with additional metadata about the rules applied. The linting results are included in the `results` property and the rules metadata is included in the `metadata` property.\n\nAlternatively, you can use the [ESLint Node.js API](../../integrate/nodejs-api) to programmatically use ESLint."
-    },
     {
         "name": "json",
         "description": "Outputs JSON-serialized results. The `json` formatter is useful when you want to programmatically work with the CLI's linting results.\n\nAlternatively, you can use the [ESLint Node.js API](../../integrate/nodejs-api) to programmatically use ESLint."
     },
     {
-        "name": "junit",
-        "description": "Outputs results to format compatible with the [JUnit Jenkins plugin](https://plugins.jenkins.io/junit/)."
+        "name": "json-with-metadata",
+        "description": "Outputs JSON-serialized results. The `json-with-metadata` provides the same linting results as the [`json`](#json) formatter with additional metadata about the rules applied. The linting results are included in the `results` property and the rules metadata is included in the `metadata` property.\n\nAlternatively, you can use the [ESLint Node.js API](../../integrate/nodejs-api) to programmatically use ESLint."
     },
```

Comment:
Changing the order of formatters in this file doesn't seem to affect the order in the output file (which I believe is how [`fs.readdirSync`](https://github.com/eslint/eslint/blob/5454c22b24f39be2dac7f28cfcfdb6c753faaf4e/Makefile.js#L457) sorts files), so it might be better to keep it the same as before.